client/finality-grandpa/src/authorities.rs

```diff
@@ -58,13 +58,14 @@ where N: Add<Output=N> + Ord + Clone + Debug,
 	}
 
 	/// Get the current set ID. This is incremented every time the set changes.
-	pub(crate) fn set_id(&self) -> u64 {
+	pub fn set_id(&self) -> u64 {
 		self.inner.read().set_id
 	}
 
 	/// Get the current authorities and their weights (for the current set ID).
-	pub(crate) fn current_authorities(&self) -> VoterSet<AuthorityId> {
-		self.inner.read().current_authorities.iter().cloned().collect()
+	pub fn current_authorities(&self) -> VoterSet<AuthorityId> {
+		// WIP: unwrap
+		VoterSet::new(self.inner.read().current_authorities.iter().cloned()).unwrap()
```

Comment:
For now let's add an expect here with a message saying that authorities should not be 0. Let's deal with this in a follow-up PR.